var/spack/repos/builtin/packages/py-pytest-pep8/package.py

```diff
@@ -9,7 +9,7 @@
 class PyPytestPep8(PythonPackage):
     """pytest plugin for efficiently checking PEP8 compliance"""
 
-    homepage = "https://bitbucket.org/pytest-dev/pytest-pep8"
+    homepage = "https://pypi.org/project/pytest-pep8/"
```

Comment:
You can remove this line, it's the default set by `pypi`

Reply:
This should be fixed.